src/core/include/rs/scene_cas.h

```diff
@@ -38,8 +38,9 @@
 #include <tf/LinearMath/Transform.h>
 
 #include <rs/types/all_types.h>
-#include <rs/conversion/conversion.h>
 #include <rs/view_names.h>
+#include <rs/conversion/conversion.h>
+
```

Comment:
revert:)

Reply:
If I revert it outputs an error: rs::conversion has not been decalared in scene_cas.h

Reply:
iinteresting...at compile time?

Reply:
so the order of the inlude matter for you? I just meant that it is not woth adding a change that has no benefit (.e.g. moving the inlcude two lines down)